src/UI/Article.jsx

```diff
@@ -75,7 +84,21 @@ function Article(props) {
         backgroundPosition: 'center',
       }}
     >
-      <div className="absolute bottom-0 left-0 right-0 h-1/2 bg-black/30 p-4 backdrop-blur-lg">
+      <a
+        href={getLink(props)}
+        target="_blank"
+        rel="noopener noreferrer"
```

Comment:
A heads-up:

For buttons or links without text, or with only icon, it's always good to add `aria-label` attribute for accessibility

[MDN: aria-label](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label)

So in this case you can do

```js
aria-label={title}
```